test/index.spec.js

```diff
@@ -0,0 +1,88 @@
+import keyboardOnlyOutlines from '../';
+
+describe('keyboard.only-outlines', () => {
+    let dispose;
+
+    beforeAll(() => {
+        const style = document.createElement('style');
+
+        style.innerText = '*:focus {outline: 1 px solid red;}';
+        document.head.append(style);
+
+        document.body.innerHTML = '<button></button>' +
```

Comment:
Do we need this? Can't we simply emit the event on `document.body`?

Reply:
We can, but I was trying to make the test more realistic by having the event be triggered as much as possible as it would in a real instance.